docs/Makefile

```diff
@@ -19,8 +19,9 @@ help:
 # Like make clean but also remove files generated by autosummary and
 # rendered videos.
 cleanall: clean
-	rm source/reference/*
-	rm -rf source/media
+	@rm source/reference/*
+	@rm -rf source/media
+	@rm -f rendering_times.csv
```

Comment:
can you make the change in `make.bat` also?

Reply:
There is not clearing section required in make.bat. Therefor I don't think changes are needed there.